lms/djangoapps/certificates/queue.py

```diff
@@ -175,7 +175,7 @@ def del_cert(self, student, course_id):
         raise NotImplementedError
 
     # pylint: disable=too-many-statements
-    def add_cert(self, student, course_id, course=None, forced_grade=None, template_file=None, generate_pdf=True):
+    def add_cert(self, student, course_id, course=None, forced_grade=None, template_file=None, generate_pdf=True):  # lint-amnesty, pylint: disable=unused-argument
```

Comment:
did not want to just remove the course param since I don't know what I am doing in this codebase.. so just adding the pylint hint for now

Reply:
I was planning on taking the same approach, this is part of some cleanup work that I have in my pipeline.